spec/seeing_is_believing_spec.rb

```diff
@@ -523,7 +523,7 @@ def meth
     end
   end
 
-  it 'does not kill parent processes' do
+  it 'does not kill parent processes', :unless => RSpec::Support::OS.windows? do
```

Comment:
The issue here is with the IChannel library and its use of UNIX sockets, and I didn't spend any time digging in to see if there was an alternative.